styles/index.css

```diff
@@ -32,17 +32,25 @@ h1 {
   padding: 15px;
 }
 
-.card button {
+.card .card-answer-btn {
```

Comment:
This should work as well, wouldn't it?

```suggestion
.card-answer-btn {
```